pages/monsters/[id].vue

```diff
@@ -290,11 +314,36 @@ const abilites = [
   modifier: formatMod(calcMod(monster[ability])),
   save: monster[`${ability}_save`] ?? calcMod(monster[ability]),
 }));
+
+const route = useRoute();
+
+const mode = ref(route.query.mode || 'normal');
+function toggleMode() {
+  switch (mode.value) {
+    case 'compact':
+      mode.value = 'normal';
+      break;
+    default:
+      mode.value = 'compact';
+      break;
+  }
+
+  navigateTo({
+    path: `/monsters/${route.params.id}`,
+    query:
+      mode.value === 'compact'
+        ? {
+            mode: 'compact',
+          }
+        : null,
+  });
+}
 </script>
 
 <style scoped lang="scss">
 .img-main {
   float: right;
+  margin-block: 1rem;
```

Comment:
Just added some vertical margin on the floated image (incidentally I noticed that the floated image gets a bit wonky at certain screen widths...)